bundles/org.openhab.binding.exec/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -69,7 +69,6 @@
 		<item-type>Switch</item-type>
 		<label>Running</label>
 		<description>Send ON to execute the command and the current state tells whether it is running or not</description>
-		<state readOnly="true"></state>
```

Comment:
Yes it makes sens to have it writable. But I would keep the file consistent and either use `<state readOnly="false"></state>` or remove the other `<state readOnly="false"></state>` from the file as well.

Reply:
Changed to have it explicitly set to `false`.